src/navigation/index.js

```diff
@@ -19,15 +19,16 @@ navigation.get = function (callback) {
 			data = data.filter(function (item) {
 				return item && item.enabled;
 			}).map(function (item) {
+				item.originalRoute = item.route;
```

Comment:
We have `.endsWith();`, could you not use that in the changes to `src/middleware/header.js` instead of saving `originalRoute`?

Reply:
I think this is a better solution. There's no chance of collissions this way. I don't see anything wrong with a little extra data.

Reply:
ok 👍